src/reducers/appReducer.js

```diff
@@ -1,7 +1,8 @@
 import { SET_MARKDOWN } from '../actions/markdownActions';
 
 export const initialState = {
-  markdown: '# hello there'
+  markdown: '# hello there',
```

Comment:
do you still need this if you have a list of markdown files? What is this state for?

Reply:
In the case of a new user coming on for the first time, this will be a default message. It will show that you type the markdown on the left, and then will get a preview on the right. I think it's good user experience.

Reply:
couldn't you put that in the array of markdowns though? and just have the first item in the array display when the user first loads the page?